feedback/static/js/src/feedback.js

```diff
@@ -42,8 +42,20 @@ function FeedbackXBlock(runtime, element) {
 		$('.feedback_thank_you', element).text(data.response);
 	    }
         });
-    }
 
+	if(vote >= 9) {
```

Comment:
Why 9? What is the magic in this number?

Reply:
Fixed to 'minVoteForFeedbackModalAppearance` const.